### PR TITLE
feat(Catalog Management): regen after recent api changes

### PR DESCRIPTION
**Pull request by benbuchanan**
## PR summary

regenerating after recent api changes

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Current vs new behavior  

adding new endpoints, removing some deprecated fields

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by benbuchanan** [comment]
```
integration
Test Suites: 20 skipped, 1 passed, 1 of 21 total
Tests:       403 skipped, 45 passed, 448 total
Snapshots:   0 total
Time:        27.536 s
Ran all test suites matching /test\/integration\//i.

example
Test Suites: 1 passed, 1 total
Tests:       45 passed, 45 total
Snapshots:   0 total
Time:        23.475 s, estimated 25 s
Ran all test suites matching /examples\/catalog-management.v1.test.js/i.
```

File: test/integration/catalog-management.v1.test.js [hunk 1]
```diff
@@ -137,6 +157,202 @@ describe('CatalogManagementV1_integration', () => {
...
   });
 
+  // Set allow publish offering
+  test('setAllowPublishOffering request example', async () => {
+    const headers = {
+      'X-Approver-Token': approverToken,
+    };
+
+    const response = await fetch(
+      `https://cm.globalcatalog.test.cloud.ibm.com/api/v1-beta/catalogs/${catalogIdLink}/offerings/${offeringIdLink}/publish/publish_approved/true`,
```
Comment (padamstx) [inline test/integration/catalog-management.v1.test.js:167]: Why are we testing these operations without the generated SDK code???

**Review by padamstx — COMMENTED** [review]
I need to understand why several operations are being tested by the integration tests and examples code where those operations are not defined in the API definition and do not appear in the generated SDK code.

**Comment by benbuchanan** [comment]
@padamstx it's the same situation we had last month in the Go SDK, these calls are only meant to be called by approvers.  In this case Partner Center calls us with an approver token, regular users can't make these calls so we don't expose them in our swagger or sdk.  But certain operations like consume publish plan require a plan to be approved and only an approver can do that.

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**